pkg/cmd/run.go

```diff
@@ -1032,15 +1032,19 @@ func writeChecksumToFile(filepath string, hash hash.Hash) error {
 	return err
 }
 
-func getSpectrumOptions(platform *v1.IntegrationPlatform, cmd *cobra.Command) spectrum.Options {
+func (o *runCmdOptions) getSpectrumOptions(platform *v1.IntegrationPlatform, cmd *cobra.Command) spectrum.Options {
 	insecure := platform.Spec.Build.Registry.Insecure
+	var stdout io.Writer
+	if o.Verbose {
+		stdout = cmd.OutOrStdout()
+	}
```

Comment:
What happen if `o.Verbose==false`? is it okey if we leave the `nil` or is it going to break in the Spectrum struct?

Reply:
Hi @squakez if it's false (default) then *stdout* stays nil and nothing will be sent to the console